magenta-lib/src/main/scala/magenta/tasks/UpdateCloudFormationTask.scala

```diff
@@ -78,6 +79,19 @@ object UpdateCloudFormationTask {
         orderedTags.map{ case (key, value) => value }.mkString("-")
     }
   }
+
+  def processTemplate(stackName: String, templateBody: String, s3Client: AmazonS3, stsClient: AWSSecurityTokenServiceClient, region: Region): Template = {
+    val templateTooBigForSdkUpload = templateBody.length > 51200
+
+    if (templateTooBigForSdkUpload) {
+      val bucketName = S3.accountSpecificBucket("riff-raff-cfn-templates", s3Client, stsClient, region, Some(1))
+      val keyName = s"$stackName-${new DateTime().getMillis}"
+      s3Client.putObject(bucketName, keyName, templateBody)
+      TemplateUrl(s"s3://$bucketName/$keyName")
+    } else {
+      TemplateBody(templateBody)
+    }
```

Comment:
why not always upload it to S3? won't be slower and we wouldn't rely on this limit of 51200 bytes that might change eventually